howso/engine/trainee.py

```diff
@@ -3066,40 +2917,39 @@ def react_into_features(
         else:
             raise ValueError("Client must have the 'react_into_features' method.")
 
-    def react_into_trainee(
+    def react_aggregate(
         self,
         *,
-        use_case_weights: bool = False,
         action_feature: Optional[str] = None,
         confusion_matrix_min_count: Optional[int] = None,
         context_features: Optional[Iterable[str]] = None,
-        contributions: Optional[bool] = None,
-        contributions_robust: Optional[bool] = None,
+        details: Optional[dict] = None,
+        feature_influences_action_feature: Optional[str] = None,
         hyperparameter_param_path: Optional[Iterable[str]] = None,
-        mda: Optional[bool] = None,
-        mda_permutation: Optional[bool] = None,
-        mda_robust: Optional[bool] = None,
-        mda_robust_permutation: Optional[bool] = None,
         num_robust_influence_samples: Optional[int] = None,
         num_robust_residual_samples: Optional[int] = None,
         num_robust_influence_samples_per_case: Optional[int] = None,
         num_samples: Optional[int] = None,
-        residuals: Optional[bool] = None,
-        residuals_robust: Optional[bool] = None,
+        prediction_stats_action_feature: Optional[str] = None,
+        residuals_hyperparameter_feature: Optional[str] = None,
+        robust_hyperparameters: Optional[bool] = None,
         sample_model_fraction: Optional[float] = None,
         sub_model_size: Optional[int] = None,
+        use_case_weights: bool = False,
         weight_feature: Optional[str] = None,
-    ):
+    ) -> DataFrame | dict:
```

Comment:
The return type should just be `DataFrame` since that is what the PandasClientMixin will return.